twindb_backup/cli.py

```diff
@@ -161,3 +162,29 @@ def restore_file(cfg, dst, backup_copy):
     except TwinDBBackupError as err:
         LOG.error(err)
         exit(1)
+
+
+@main.group('verify')
+@PASS_CFG
+def verify(cfg):
+    """Verify backup"""
+    LOG.debug('Restore: %r', cfg)
+
+
+@verify.command('mysql')
+@click.argument('backup_copy', required=False)
+@click.option('--dst', help='Directory where to restore the backup copy',
+              default='.', show_default=True)
+@click.option('--hostname', help='Verify where backup was made',
```

Comment:
If backup_copy is `latest` this option specifies hostname where the backup copy was taken.